### PR TITLE
Unexported solvers

### PR DESCRIPTION
**Pull request by joaquimg**
Add support to unexported `XpressSolver` now `Xpress.XpressSolver`

Is it what you meant [here](https://github.com/JuliaOpt/MathOptInterface.jl/issues/70#issuecomment-330079082) @mlubin ?

Should I do something elsewhere?

### CODE REVIEW

**Comment by mlubin** [comment]
There's also http://www.juliaopt.org/JuMP.jl/0.18/installation.html#getting-solvers. You can modify the HTML directly and do a PR to the `gh-pages` branch.

File: docs/src/installation.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Solver support in Julia is currently provided by writing a solver-specific packa
...
 | [CPLEX](http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/)   | [CPLEX.jl](https://github.com/JuliaOpt/CPLEX.jl)                                | `CplexSolver()`           |  Comm.      | X     | X     | X     |       |       |       |
 | [ECOS](https://github.com/ifa-ethz/ecos)                                         | [ECOS.jl](https://github.com/JuliaOpt/ECOS.jl)                                  | `ECOSSolver()`            |  GPL        | X     | X     |       |       |       |       |
-| [FICO Xpress](http://www.fico.com/en/products/fico-xpress-optimization-suite)    | [Xpress.jl](https://github.com/JuliaOpt/Xpress.jl)                              | `XpressSolver()`          |  Comm.      | X     | X     | X     |       |       |       |
+| [FICO Xpress](http://www.fico.com/en/products/fico-xpress-optimization-suite)    | [Xpress.jl](https://github.com/JuliaOpt/Xpress.jl)                              | `Xpress.XpressSolver()`          |  Comm.      | X     | X     | X     |       |       |       |
```
Comment (mlubin) [inline docs/src/installation.md]: Have you tested that the alignment is correct and the documentation compiles correctly?

Comment (joaquimg) [reply docs/src/installation.md]: alignment looks good on #1104

Comment (mlubin) [reply docs/src/installation.md]: #1104 does not test this. You have to build a local copy of the docs by following https://github.com/JuliaOpt/JuMP.jl/tree/master/docs#jumpjl-documentation-readme

**Comment by mlubin** [comment]
Now that I think about it, I'd prefer no more commits to master that will create conflicts with the MOI branch. This change can be made against `release-0.18`.

**Comment by joaquimg** [comment]
I agree, much better there.

**Comment by joaquimg** [comment]
Should I leave this open?

**Comment by mlubin** [comment]
Let's leave it open as a reminder. The development docs aren't updated either (but shouldn't be yet).

**Pull request closed without merging**